### PR TITLE
fix: use correct command for merging

### PR DESCRIPTION
**Pull request by iamsobanjaved**
- Also mention variables from Github env because of better verbosity.
- Run job on demand with `workflow_dispatch` for better debugging.

### CODE REVIEW

**Review by UsamaSadiq — APPROVED** [review]

**Comment by feanil** [comment]
Isn't this workflow just here to be referenced by [the template](https://github.com/openedx/.github/blob/master/workflow-templates/upgrade-python-requirements.yml) which is what should exist in all the other repos?  The template already allows for running via `workflow_dispatch` shouldn't that be sufficient?

**Comment by awais786** [comment]
> the template

its a `upgrade-python-requirements.yml` but here we are trying to run other work flow `merge-python-requirements-upgrade-prs.yml`.

**Comment by iamsobanjaved** [comment]
This PR is still in progress

**Comment by iamsobanjaved** [comment]
@feanil no this isn't to be used by the template. Instead, this creates a recurring action to merge all eligible upgrade PRs. And it is ready to be merged.

**Comment by feanil** [comment]
Oh, I see, you're planning on running it once in this repo and having it merge all PRs across the org.  I didn't understand that from just reading the workflow.  Maybe it makes sense to add a comment at the top about how you expect this workflow to be used?

**Comment by iamsobanjaved** [comment]
@feanil added a comment on top to explain the workflow

**Review by feanil — APPROVED** [review]

**Comment by feanil** [comment]
Let me know when y'all think this is ready to merge.

File: .github/workflows/merge-python-requirements-upgrade-prs.yml
Comment (iamsobanjaved) [reply .github/workflows/merge-python-requirements-upgrade-prs.yml:3]: I had this concern in my mind initially but then got approval so thought it is fine to go with it as it is. I can write a workflow in the `edx/.github` repo that will re-use this one. Can we merge this and come up with a new PR with the above-mentioned change so in meantime we can test whether this change is working or not?

Comment (feanil) [reply .github/workflows/merge-python-requirements-upgrade-prs.yml:3]: Absolutely, are you ready for this to be merged now?

Comment (iamsobanjaved) [reply .github/workflows/merge-python-requirements-upgrade-prs.yml:3]: Yes, you can go ahead and merge this.

**Pull request merged**
